library/core/src/future/pending.rs

```diff
@@ -9,8 +9,6 @@ use crate::task::{Context, Poll};
 ///
 /// This `struct` is created by the [`pending`] function. See its
```

Comment:
While we're improving things:
```suggestion
/// This `struct` is created by [`pending()`]. See its
```